cmd/integrations_tfc.go

```diff
@@ -0,0 +1,148 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"connectrpc.com/connect"
+	"github.com/overmindtech/sdp-go"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// createTfcCmd represents the tfc command
+var createTfcCmd = &cobra.Command{
+	Use:    "create-tfc",
+	Short:  "Initialize the HCP Terraform Cloud integration",
+	Long:   "Create the initial set of parameters to configure HCP Terraform to talk to Overmind.",
+	PreRun: PreRunSetup,
+	RunE:   CreateTfc,
+}
+
+// getTfcCmd represents the tfc command
+var getTfcCmd = &cobra.Command{
+	Use:    "get-tfc",
+	Short:  "Retrieve the existing parameters for the HCP Terraform Cloud integration",
+	Long:   "Retrieve the existing parameters for the HCP Terraform Cloud integration.",
+	PreRun: PreRunSetup,
+	RunE:   GetTfc,
+}
+
+// deleteTfcCmd represents the tfc command
+var deleteTfcCmd = &cobra.Command{
+	Use:    "delete-tfc",
+	Short:  "Delete the HCP Terraform Cloud integration",
+	Long:   "This will delete the HCP Terraform Cloud integration and disable all access from HCP Terraform Cloud to Overmind.",
+	PreRun: PreRunSetup,
+	RunE:   DeleteTfc,
+}
+
+func init() {
+	integrationsCmd.AddCommand(createTfcCmd)
+	integrationsCmd.AddCommand(getTfcCmd)
+	integrationsCmd.AddCommand(deleteTfcCmd)
+
+	addAPIFlags(createTfcCmd)
+	addAPIFlags(getTfcCmd)
+	addAPIFlags(deleteTfcCmd)
+
+	// Here you will define your flags and configuration settings.
+
+	// Cobra supports Persistent Flags which will work for this command
+	// and all subcommands, e.g.:
+	// tfcCmd.PersistentFlags().String("foo", "", "A help for foo")
+
+	// Cobra supports local flags which will only run when this command
+	// is called directly, e.g.:
+	// tfcCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+}
+
+func CreateTfc(cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+
+	ctx, oi, _, err := login(ctx, cmd, []string{"config:write", "api_keys:write", "changes:write", "explore:read", "request:send", "reverselink:request"})
```

Comment:
should  "api_keys:write" be added, if you are making sure they have everything ?

Reply:
it's right there? What am I missing?